README.md

```diff
@@ -1,2 +1,7 @@
 # ldap-tgen
 An LDAP (Lightweight Directory Access Protocol) traffic generator. It can be used for ldap server load testing.
+
+
+## License
+
+This project is licensed under the MPL-2.0 license - see the [LICENSE](https://github.com/nokia/ldap-tgen/blob/master/LICENSE).
```

Comment:
hi gergely,

wouldn't it be more elegant to use some kind of [relative link](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file)?

thank you,

yohann